libs/database/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
```

Comment:
FYI: Vi har allerede et database-lib for database: [etterlatte-database](https://github.com/navikt/pensjon-etterlatte-saksbehandling/tree/main/libs/etterlatte-database) :)

Reply:
Fikset nå